docs/external_access.md

```diff
@@ -0,0 +1,89 @@
+## Enabling External Access to MongoDB deployment
+
+This guide assumes that the operator is installed and a MongoDB deployment is yet to be done but you have a chosen namespace that you are installing into. We will install cert-manager and then generate certificates and configure split-horizon to support internal and external DNS names for configuring external access to the replicaset.
+
+### Install cert-manager
+
+```sh
+kubectl create namespace cert-manager
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
```

Comment:
[nit] do we need this if we specified `kubectl create namespace cert-manager` as the first command?

Reply:
Good point. Resolved in latest commit.